django/forms/models.py

```diff
@@ -93,10 +93,21 @@ def model_to_dict(instance, fields=None, exclude=None):
     return data
 
 
+def apply_limit_choices_to_to_formfield(formfield):
+    """
+    Apply limit_choices_to to the formfield's queryset if it and the
```

Comment:
I think "if needed" rather than "if it and the....exist." is sufficient given the code is straightforward.